tests/pods/pods.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -e
+#
+# Script to run Sysbox-PODs integration testcases in the supported platforms --
+# only Ubuntu for now.
+#
+
+# Returns linux distro running in the system.
+function get_host_distro() {
+	local distro=$(cat /etc/os-release | awk -F"=" '/^ID=/ {print $2}' | tr -d '"')
+	echo $distro
+}
+
+function main() {
+
+	# Argument testName is optional.
+	if [ $# -eq 1 ]; then
+		printf "\nExecuting $1 ... \n"
+		bats --tap $1
+		exit 0
+	fi
+
+	local distro=$(get_host_distro)
+
+	# Skip sysbox-pod testcases in all the unsupported distros.
+	if [[ ${distro} != "ubuntu" ]]; then
```

Comment:
My goal with this approach is to allow all automation jobs to succeed without touching the individual test-cases being executed.

If we were to add these changes within bats itself, then we wouldn't be able to execute individual bats testcases over 'suspicious' distros. This is precisely the reason i'm honoring the desired testcase to execute in these shell wrappers, even if they are likely to fail, so that we have a way to troubleshoot/verify that/if problem exists at any given time.

Reply:
If the test is not expected to work on a particular distro, that info should be in the test itself, rather than in the wrapper shell script. Otherwise if we run the test on say Fedora with `bats -t tests/pods/basis.bats` and if fails, we will spend cycles figuring out why it failed, because there is nothing in the test telling us that it's expected to fail in Fedora.

Reply:
Ok. I just changed it.